contents/articles/2018-01_pupil_community/index.md

```diff
@@ -0,0 +1,64 @@
+--- 
+title: Pupil Community
+date: Fri Jan 12 2018 14:08:00 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Pupil is a community driven development project. Check out what the community has been doing with Pupil..."
+featured_img: ""
+featured_img_thumb: ""
+---
+
+Happy 2018!
+Our first blog post of the year will be about the Pupil community, looking at how much Pupil has grown!
+Pupil is a community driven development project to make eye tracking hardware that is accessible, hackable, affordable and software that is open source.
+
+Check out what the community has been doing! Here are a list of custom tools and projects built with Pupil.
```

Comment:
add something to state that this is by no means complete and there may be many more out there!

Reply:
Yes, i will make the change.